flyteidl2/task/task_definition.proto

```diff
@@ -121,6 +124,20 @@ message Task {
   TaskMetadata metadata = 2 [(buf.validate.field).required = true];
 }
 
+// Link to source code used to define this entity
+message SourceCode {
+  string link = 1;
+}
+
+message DescriptionEntity {
```

Comment:
I really don't like this name but I can't think of anything better.  cc @pingsutw any ideas?

Reply:
Documentation?